docs/user/integration/dynatrace/README.md

```diff
@@ -151,7 +153,9 @@ kubectl -n $DYNATRACE_NS create secret generic dynatrace-token --from-literal="a
 
 ### Ingest Metrics (Experimental)
 
-1. Deploy the TracePipeline and replace the `{ENVIRONMENT_ID}` placeholder with the environment Id of Dynatrace SaaS:
+To collect custom metrics, you usually use the Dynatrace annotation approach, as the Dynatrace OTLP integration is [limited](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/metrics/ingest/migration-guide-otlp-exporter#migrate-collector-configuration). As long as your workload is conform to the limitations (not exporting histograms, using delta aggregation temporality), you can leverage the metric functionality to push OTLP metrics centralized to Dynatrace. Hereby, the prometheus feature of the MetricPipeline will not work as it will hit the limitations by design.
```

Comment:
```suggestion
To collect custom metrics, you usually use the Dynatrace annotation approach, because the Dynatrace OTLP integration is [limited](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/metrics/ingest/migration-guide-otlp-exporter#migrate-collector-configuration). As long as your workload is conform to the limitations (not exporting histograms, using delta aggregation temporality), you can use the metric functionality to push OTLP metrics to Dynatrace. In this case, the Prometheus feature of the MetricPipeline does not work because it hits the limitations by design.
```